doc/install.txt

```diff
@@ -4,891 +4,17 @@
 Installing Theano
 =================
 
+On the following pages you will find different installation instructions depending
```

Comment:
If the purpose is still to make all this clutter-free I think this sentence is totally unnecessary?